clockify_client/models/task.py

```diff
@@ -66,14 +65,14 @@ def get_tasks(
         :param params        Request URL query parameters.
         :return              List with dictionaries with task object representation.
         """
-        _url = f"{self.base_url}/workspaces/{workspace_id}/projects/{project_id}"
+        _url = f"/workspaces/{workspace_id}/projects/{project_id}"
```

Comment:
I'd rename this base_path rather than _url

Reply:
Fixed